pages/common/go-fmt.md

```diff
@@ -0,0 +1,24 @@
+# go fmt
+
+> Format Go source files.
+> More information: <https://pkg.go.dev/cmd/go#hdr-Gofmt__reformat__package_sources>.
+
+- Format Go source files in the current directory:
+
+`go fmt`
+
+- Format a specific Go package:
+
+`go fmt {{package}}`
+
+- Format the package in the current directory and all subdirectories (note the `...`):
+
+`go fmt ./...`
```

Comment:
This requires token syntax:
```suggestion
`go fmt {{./...}}`
```

Reply:
thanks for pointing that out.  I updated the commit with this change.